web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -52,19 +52,15 @@ export class GridComponentRenderer {
     )
   }
 
-  public predictionItemValueSetter = (
-    params: Pick<GridValueSetterParams, 'row' | 'value'>,
-    field: string,
-    precision: number
-  ) => {
+  public predictionItemValueSetter = (params: Pick<GridValueSetterParams, 'row' | 'value'>, field: string) => {
     const oldValue = params.row[field].value
     const newValue = Number(params.value)
 
     if (isNaN(oldValue) && isNaN(newValue)) {
       return { ...params.row }
     }
     // Check if the user has edited the value. If so, update the value and choice to reflect the Manual edit.
-    if (newValue.toFixed(precision) !== params.row[field].value.toFixed(precision)) {
+    if (newValue !== params.row[field].value) {
```

Comment:
The intention of the original (poorly implemented by me) code was to allow a user to 'cancel' editing a cell without us marking the cell as manually edited. The data we're getting from the API typically has many decimal places, but we show 0 or 1 decimal places in the UI. If you enter editing mode in a cell and cancel, the valueSetter sees the rounded version which gets compared to the original value with many decimal places. This comparison on line 63 will now always evaluate to `true` and a cell will always be marked as manually edited. At the time I was trying to emulate the default behaviour of the datagrid. Probably not a big deal though.

Reply:
Wow, thanks!